apps/molgenis-components/src/components/tables/cellTypes/FileDisplay.vue

```diff
@@ -1,5 +1,9 @@
 <template>
-  <a v-if="data.id" :href="data.url">
+  <a
+    v-if="data.id"
+    :href="data.url"
+    :target="data.extension.toLowerCase() == 'html' ? '_blank' : null"
```

Comment:
does _blank also work for images? I.e. I would propose that we always do '_blank' or does that result in weird situations?